test/specs/anInitialSuite.e2e.ts

```diff
@@ -91,59 +103,68 @@ describe('An Initial Suite', async () => {
     await testSetup.createProject('Developer');
   });
 
-  step('Verify our extensions are loaded after creating an SFDX project', async () => {
-    const workbench = await browser.getWorkbench();
-    await utilities.runCommandFromCommandPrompt(workbench, 'Developer: Show Running Extensions', 2);
-
-    let matchesFound = 0;
-    const extensionNameDivs = await $$('div.name');
-    for (const extensionNameDiv of extensionNameDivs) {
-      const text = await extensionNameDiv.getText();
-
-      if (text.startsWith('salesforce.salesforcedx-vscode-')) {
-        matchesFound++;
-        utilities.log(`AnInitialSuite - extension ${text} is loaded`);
+  step(
+    'Verify our extensions are loaded after creating an SFDX project',
+    async () => {
+      const workbench = await browser.getWorkbench();
+      await utilities.runCommandFromCommandPrompt(
+        workbench,
+        'Developer: Show Running Extensions',
+        2
+      );
+
+      // Close panel so the visualforce extension can be seen
+      await utilities.runCommandFromCommandPrompt(
+        workbench,
+        'View: Close Panel',
```

Comment:
@jeffb-sfdc I added this here to close the output panel so we can verify all 7 extensions :)

Reply:
Nice!  All are displayed on my monitor, but several other extension also appear, so this might flap in GitHub, but for the moment I say we keep what you have.